source/dlangbot/github.d

```diff
@@ -262,10 +269,10 @@ void searchForAutoMergePrs(string repoSlug)
     static immutable labels = ["auto-merge", "auto-merge-squash"];
     foreach (issue; getIssuesForLabels(repoSlug, labels))
     {
-        if (!issue.isPullRequest)
+        if (!issue.isPullRequest) // TODO: query is:pr
             continue;
 
-        auto pr = issue.toPullRequest;
+        auto pr = issue.pullRequest;
```

Comment:
The reason for `toPullRequest` was to save API requests. This is in the CRON loop and will trigger a new request _for each_ PR here.

Reply:
Yes, but not avoidable if we want to check the status. Those are only open PRs with auto-merge btw, a rather small number.

open auto-merges:
[dmd](https://github.com/dlang/dmd/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 1
[druntime](https://github.com/dlang/druntime/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 1
[phobos](https://github.com/dlang/phobos/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 1
[dlang.org](https://github.com/dlang/dlang.org/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 1
[installer](https://github.com/dlang/installer/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 0
[tools](https://github.com/dlang/tools/pulls?q=is%3Aopen+is%3Apr+label%3Aauto-merge) -> 0

And that's only because of the merge_stable step.